api/config.py

```diff
@@ -162,15 +162,25 @@ class Configuration(CoreConfiguration):
         {
             "key": COLOR_SCHEME,
             "label": _("Color scheme"),
-            "description": _("This tells clients what colors to use when rendering this library's OPDS feed."),
+            "description": _("This tells clients what color scheme to use when rendering this library's OPDS feed."),
             "options": [
-                { "key": "blue", "label": _("Blue") },
-                { "key": "red", "label": _("Red") },
-                { "key": "gray", "label": _("Gray") },
-                { "key": "gold", "label": _("Gold") },
-                { "key": "green", "label": _("Green") },
-                { "key": "teal", "label": _("Teal") },
-                { "key": "purple", "label": _("Purple") },
+                dict(key="amber", label=_("Amber")),
+                dict(key="black", label=_("Black")),
+                dict(key="blue", label=_("Blue")),
+                dict(key="bluegrey", label=_("Blue Grey")),
+                dict(key="brown", label=_("Brown")),
+                dict(key="cyan", label=_("Cyan")),
+                dict(key="darkorange", label=_("Dark Orange")),
+                dict(key="darkpurple", label=_("Dark Purple")),
+                dict(key="green", label=_("Green")),
+                dict(key="grey", label=_("Grey")),
```

Comment:
You changed the spelling of gray here and thus would probably need a migration for anyone who already picked it. Also, why not store these values as hex? Do we expect to change the exact values in the future?

Reply:
Hex would also make it possible to show a preview of the color in the admin interface.

Reply:
They don't correspond to exact values even now: Android uses colors that are more saturated than iOS uses. In the future, they won't necessarily correspond to a single color -- there could be a normal color and a highlight color for a given color scheme.

Reply:
Accounts.json currently has a hex value though, and it's the same across platforms, so they at least use the same base value to generate their color schemes.